net-proxy/sqtop/sqtop-2015.02.08.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="$(ver_rs 1-2 -)"
+
+DESCRIPTION="Squid Top - top for Squid"
+HOMEPAGE="https://github.com/paleg/sqtop"
+SRC_URI="https://github.com/paleg/sqtop/archive/v${MY_PV}.tar.gz"
```

Comment:
`-> ${P}.tar.gz` for an identifiable distfile.

Reply:
added locally.  i'm able to reproduce ncurses but haven't dug into it yet.